test/src/test/java/org/apache/rocketmq/test/container/ContainerIntegrationTestBase.java

```diff
@@ -186,6 +187,7 @@ protected static void createTopicTo(BrokerController masterBroker, String topicN
         try {
             TopicConfig topicConfig = new TopicConfig(topicName, rqn, wqn, 6, 0);
             defaultMQAdminExt.createAndUpdateTopicConfig(masterBroker.getBrokerAddr(), topicConfig);
+            System.out.println("Create topic " + topicName + " to " + masterBroker.getBrokerIdentity().getCanonicalName() + "@" + masterBroker.getBrokerAddr());
```

Comment:
We should remove the useless System.out.println

Reply:
done